jetty-server/src/main/config/etc/jetty-acceptratelimit.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0"?>
+<!DOCTYPE Configure PUBLIC "-//Jetty//Configure//EN" "http://www.eclipse.org/jetty/configure_9_3.dtd">
+<Configure id="Server" class="org.eclipse.jetty.server.Server">
+  <Call name="addBean">
+    <Arg>
+      <New class="org.eclipse.jetty.server.AcceptRateLimit">
+        <Arg name="maxRate" type="int"><Property name="jetty.acceptrate.limit" default="1000" /></Arg>
+        <Arg name="period" type="long"><Property name="jetty.acceptrate.period" default="1000" /></Arg>
+        <Arg name="units"><Call class="java.util.concurrent.TimeUnit" name="valueOf"><Arg>MILLISECONDS</Arg></Call></Arg>
```

Comment:
Remove the unit parameter, assume milliseconds.

Reply:
Nah!  Now we are increasingly using nano time, the possibility of confusion of just assuming milliseconds is too high.   passing `long,timeUnit` is an established pattern to avoid confusion and it's only on configuration, so no cost.